PullRequestR/android/build.gradle

```diff
@@ -0,0 +1,30 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.0'
+        
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        maven {
+            url 'https://www.myget.org/F/ping-verify/maven/'
```

Comment:
Drop the patched feed and just use the production build. It may not work until the service deployment, but we should avoid using the patched version in an official sample :)

Reply:
Yeah, we need to loop back with the service team to make sure that the fix for receiving pings went in.

Reply:
checked with service team this evening